src/MyMind.Web/Controllers/TestController.cs

```diff
@@ -14,8 +14,22 @@ public class TestController : BaseApiController
         private readonly ISessionFactory _sessionFactory;
 
         public TestController(ISessionFactory sessionFactory)
+            => this._sessionFactory = sessionFactory;
+
+        [HttpDelete]
```

Comment:
Metody w TestController robiłbym jako http get
Chodzi tutaj o to, żeby dostęp do nich był możliwie najprostszy
Http get możemy wykonać wpisując url w przeglądarce

Reply:
zrobiłem HttpDelete, ponieważ był taki ładny czerowny kolor :)
zmieniłem na HttpGet